src/Microsoft.Health.Fhir.Core/Features/Search/Converters/StringNodeToTokenSearchValueTypeConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Hl7.Fhir.ElementModel;
+using Microsoft.Health.Fhir.Core.Features.Search.SearchValues;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Converters
+{
+    /// <summary>
+    /// A converter used to convert from <see cref="string"/> to a list of <see cref="TokenSearchValue"/>.
+    /// </summary>
+    public class StringNodeToTokenSearchValueTypeConverter : FhirNodeToSearchValueTypeConverter<TokenSearchValue>
```

Comment:
Also until we remove them (hopefully in soon), we should implement this in the old converters too:
https://github.com/microsoft/fhir-server/tree/master/src/Microsoft.Health.Fhir.Shared.Core/Features/Search/Converters

Reply:
Thanks for the direction. Added them in eaea80f.